mne/forward/tests/test_field_interpolation.py

```diff
@@ -196,7 +193,7 @@ def test_make_field_map_meeg():
 def _setup_args(info):
     """Helper to test_as_meg_type_evoked."""
     coils = _create_meg_coils(info['chs'], 'normal', info['dev_head_t'])
-    int_rad, noise, lut_fun, n_fact = _setup_dots('fast', coils, 'meg')
```

Comment:
There isn't really any speed difference, so we might as well get better results. I can revert if it's a problem